CHANGELOG.md

```diff
@@ -78,6 +78,7 @@
 * [ENHANCEMENT] Include tempo-cli in the release [#1086](https://github.com/grafana/tempo/pull/1086) (@zalegrala)
 * [ENHANCEMENT] Add search on span status [#1093](https://github.com/grafana/tempo/pull/1093) (@mdisibio)
 * [ENHANCEMENT] Slightly improved compression performance [#1094](https://github.com/grafana/tempo/pull/1094) (@bboreham)
+* [ENHANCEMENT] Add middleware to compress frontend HTTP responses with gzip if requested [#1080](https://github.com/grafana/tempo/pull/1080) (@zalegrala)
```

Comment:
Please move this into the _main / unreleased_ section. Otherwise this will be added to v1.2.0.

Reply:
Oh sorry, I missed that.